framework/src/bundle/BundlePrivate.cpp

```diff
@@ -826,23 +826,24 @@ namespace cppmicroservices
             {
                 sr.Unregister();
             }
-            catch (std::logic_error const& /*ignore*/)
+            catch (std::logic_error const& ex/*ignore*/)
```

Comment:
its odd to me that the linter flagged `catch (std::logic_error const& /*ignore*/)`. Isn't this a best practice when not using the exception object in the catch clause?